test/test_endpoints.py

```diff
@@ -352,3 +392,39 @@ def test_get_transaction_trace_of_rejected():
     resp_body = resp.json()
     assert resp_body["code"] == str(StarknetErrorCode.NO_TRACE)
     assert resp.status_code == 500
+
+
+@devnet_in_background()
+def test_get_transaction_with_tx_hash_0():
+    """Should fail on get_transaction with hash 0 without 0x prefix"""
+    resp = get_transaction("0")
+    resp_body = resp.json()
+    assert resp_body["message"].startswith(INVALID_TRANSACTION_HASH_MESSAGE)
```

Comment:
Why `startswith` and not equality?

Reply:
there is ` got: '{arg}'` after message so I used `startswith`